### PR TITLE
Node test

### PR DESCRIPTION
**Pull request by kamathis4**

### CODE REVIEW

File: keras_core/ops/node_test.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        self.assertEqual(node1.is_input, True)
+        self.assertEqual(node2.is_input, False)
+
+        self.assertEqual(node1.operation, a_layer)
+        self.assertEqual(node2.operation, b_layer)
+
+        self.assertEqual(node1.output_tensors[0], a)
+        self.assertEqual(node1.output_tensors[0].shape, shape)
+
+        self.assertEqual(a_layer.__dict__["_inbound_nodes"][0], node1)
```
Comment (fchollet) [inline keras_core/ops/node_test.py:44]: This is just `a_layer. _inbound_nodes`, so let's use that?

File: keras_core/ops/node_test.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        a = KerasTensor(shape=shape)
+        a_layer = DummyLayer()
+        node = Node(a_layer, outputs=a, call_args=(), call_kwargs={})
+
+        self.assertEqual(node.is_input, True)
+
+        self.assertEqual(node.output_tensors[0], a)
+        self.assertEqual(node.output_tensors[0].shape, shape)
+
+    def test_single_wired_layers(self):
```
Comment (fchollet) [inline keras_core/ops/node_test.py:25]: Please add a comment describing what this case is testing

File: keras_core/ops/node_test.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        shape = (2, 3, 4)
+        a = KerasTensor(shape=shape)
+        a_layer = DummyLayer()
+        node1 = Node(a_layer, outputs=a, call_args=(), call_kwargs={})
+
+        b = KerasTensor(shape=shape)
+        b_layer = DummyLayer()
+        node2 = Node(b_layer, outputs=b, call_args=(a,), call_kwargs={})
```
Comment (fchollet) [inline keras_core/ops/node_test.py:33]: We should also test kwargs as well

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by kamathis4** [comment]
Closing PR because there seems to be an issue with a wrong commit and will add another PR

**Pull request closed without merging**